addin-vesting/cli/src/main.rs

```diff
@@ -1,227 +1,399 @@
+    rpc_client: RpcClient,
+    governance_program_id: Pubkey,
+    vesting_addin_program_id: Pubkey,
+    payer: Keypair,
+    vesting_owner: Keypair,
+    vesting_token_pubkey: Pubkey,
+    mint_pubkey: Pubkey,
+    realm_pubkey: Pubkey,
+    new_vesting_owner_pubkey: Pubkey,
+) {
+
+    let change_owner_instruction = change_owner_with_realm(
+        &vesting_addin_program_id,
+        &vesting_token_pubkey,
+        &vesting_owner.pubkey(),
+        &new_vesting_owner_pubkey,
+        &governance_program_id,
+        &realm_pubkey,
+        &mint_pubkey,
+    )
+    .unwrap();
+
+    let mut transaction = Transaction::new_with_payer(&[change_owner_instruction], Some(&payer.pubkey()));
+
+    let latest_blockhash = rpc_client.get_latest_blockhash().unwrap();
+    transaction.sign(
+        &[&payer, &vesting_owner],
+        latest_blockhash,
+    );
+
+    rpc_client.send_transaction(&transaction).unwrap();
+}
+
+fn command_create_voter_weight_record(
+    rpc_client: RpcClient,
+    governance_program_id: Pubkey,
+    vesting_addin_program_id: Pubkey,
+    payer: Keypair,
+    record_owner_pubkey: Pubkey,
+    mint_pubkey: Pubkey,
+    realm_pubkey: Pubkey,
+) {
+
+    let instruction = create_voter_weight_record(
+        &vesting_addin_program_id,
+        &record_owner_pubkey,
+        &payer.pubkey(),
+        &governance_program_id,
+        &realm_pubkey,
+        &mint_pubkey,
+    )
+    .unwrap();
+
+    let mut transaction = Transaction::new_with_payer(&[instruction], Some(&payer.pubkey()));
+
+    let latest_blockhash = rpc_client.get_latest_blockhash().unwrap();
+    transaction.sign(
+        &[&payer],
+        latest_blockhash,
+    );
+
+    rpc_client.send_transaction(&transaction).unwrap();
+}
+
+fn command_set_vote_percentage_with_realm(
+    rpc_client: RpcClient,
+    governance_program_id: Pubkey,
+    vesting_addin_program_id: Pubkey,
+    payer: Keypair,
+    vesting_authority: Keypair,
+    vesting_owner_pubkey: Pubkey,
+    vesting_token_pubkey: Pubkey,
+    mint_pubkey: Pubkey,
+    realm_pubkey: Pubkey,
+) {
+
+    let instruction = set_vote_percentage_with_realm(
+        &vesting_addin_program_id,
+        &vesting_token_pubkey,
+        &vesting_owner_pubkey,
+        &vesting_authority.pubkey(),
+        &governance_program_id,
+        &realm_pubkey,
+        &mint_pubkey,
+        70*100,
     )
     .unwrap();
 
-    let mut transaction = Transaction::new_with_payer(&[unlock_instruction], Some(&payer.pubkey()));
+    let mut transaction = Transaction::new_with_payer(&[instruction], Some(&payer.pubkey()));
 
-    let recent_blockhash = rpc_client.get_recent_blockhash().unwrap().0;
+    let latest_blockhash = rpc_client.get_latest_blockhash().unwrap();
     transaction.sign(
-        &[&payer, &destination_token_account_owner],
-        recent_blockhash,
+        &[&payer,&vesting_authority],
+        latest_blockhash,
     );
 
     rpc_client.send_transaction(&transaction).unwrap();
 }
 
 fn command_info(
```

Comment:
I think it will be useful to have a command to display all vesting by a specific owner.